### ISSUE TITLE
0.8.0 not work new version. black screen(keep loading) and game stop

### ISSUE THREAD

**Issue by Twotoolus-FLY-LShst** [xforce/anno1800-mod-loader#95]
0.8.0 not work new version. black screen(keep loading) and game stop

**Comment by xforce** [comment]
Not seeing any issues on my end. Could you share the `mod-loader.log` file?

**Comment by Twotoolus-FLY-LShst** [comment]
> Not seeing any issues on my end. Could you share the `mod-loader.log` file?

where to get log file?

i check games\Anno 1800\Bin\Win64, games\Anno 1800\mods
no log file

**Comment by xforce** [comment]
> > Not seeing any issues on my end. Could you share the `mod-loader.log` file?
> 
> where to get log file?
> 
> i check games\Anno 1800\Bin\Win64, games\Anno 1800\mods
> no log file

`games\Anno 1800\logs`

**Comment by camp76** [comment]
hi....i tested it today with one mod.....it doesn`t work for me.....pls can u help me,to find the mistake.i send a mod-loader.log....nice greetings 
[mod-loader.log](https://github.com/xforce/anno1800-mod-loader/files/7086445/mod-loader.log)

**Comment by Daywalkerhawk** [comment]
it Says not Compatible with this game version
[mod-loader.log](https://github.com/xforce/anno1800-mod-loader/files/7087515/mod-loader.log)

**Comment by xforce** [comment]
I changed a few things that hopefully fix this.
Since I can't really reproduce this locally here is a build to try it out 
https://cdn.discordapp.com/attachments/585948301698793475/882402462122790943/loader.zip

**Comment by Daywalkerhawk** [comment]
the Game loads now but there is no Modified tag in the Main Menu witch means no mods are active

**Comment by xforce** [comment]
> the Game loads now but there is no Modified tag in the Main Menu witch means no mods are active

No, in Game Update 12 Ubisoft broke or disabled, I don't know which one, the Modding Detected message.
So even if mods are loaded and working you won't see 'Modding Detected' in the Main Menu anymore.

**Comment by Daywalkerhawk** [comment]
yes your right i just loaded my game and can confirm it works testet it with free farm placement.

**Comment by MrDimitrius** [comment]
Hi! I’ve installed new version and it doesn’t load some of my mods, is there any issues with this new mod loader version? Should I wait for update? Thank you!

**Comment by Twotoolus-FLY-LShst** [comment]
> I changed a few things that hopefully fix this.
> Since I can't really reproduce this locally here is a build to try it out
> https://cdn.discordapp.com/attachments/585948301698793475/882402462122790943/loader.zip

i download, and try game start.
still keep loaing(black screen) and game shut down.

if not mod install(inside delete all file in mod folder), game can load main menu
i try too verifying check, and pasting your new version. same not work _(

log same

[2021-09-01 13:25:19.467] [info] Start applying xml operations
[2021-09-01 13:25:20.334] [info] Finished applying xml operations


thank you keep support mod =D

**Comment by MrDimitrius** [comment]
Unfortunately that didn’t help…If you could try running this addon - Pellets Power Plant https://www.nexusmods.com/anno1800/mods/49?tab=description and figure out why it doesn’t work with this new update, I’ll appreciate it Thank you! Sent from Mail for Windows From: TwotoolusFLY LSh.stSent: Wednesday, September 1, 2021 12:42 AMTo: xforce/anno1800-mod-loaderCc: MrDimitrius; CommentSubject: Re: [xforce/anno1800-mod-loader] 0.8.0 not work new version. black screen(keep loading) and game stop (#95) I changed a few things that hopefully fix this.Since I can't really reproduce this locally here is a build to try it outhttps://cdn.discordapp.com/attachments/585948301698793475/882402462122790943/loader.zipi download, and try game start.still keep loaing(black screen) and game shut down.if not mod install(inside delete all file in mod folder), game can load main menui try too verifying check, and pasting your new version. same not work _(log same[2021-09-01 13:25:19.467] [info] Start applying xml operations[2021-09-01 13:25:20.334] [info] Finished applying xml operationsthank you keep support mod =D—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by benny741** [comment]
black screen and game shut down here.

[mod-loader.log](https://github.com/xforce/anno1800-mod-loader/files/7088702/mod-loader.log)

**Comment by chihsun** [comment]
if any mods-file in /mods, it crashed, and show nothing in logs.
if no mods-file in /mods, it started normally with logs, below
```
[2021-09-01 17:43:22.875] [info] Start applying xml operations
[2021-09-01 17:43:22.876] [info] Finished applying xml operations
```

**Comment by Weber-Ruc** [comment]
My game keeps crashing with any mod in /mods.
[mod-loader.log](https://github.com/xforce/anno1800-mod-loader/files/7090174/mod-loader.log)

**Comment by jaygotony** [comment]
When it has no mods in the mods folder, game could started up.^^ 
With installed  0.8.0 mod Loader.

But, if I wrote an assets.xml put it in the location as below. 
Then it crash.

ANNO1800\mods\TEST\data\config\export\main\asset

I wrote only

```
<ModOps>
</ModOps>
```

Language Setting: Chinese Traditional 

[mod-loader.log](https://github.com/xforce/anno1800-mod-loader/files/7092984/mod-loader.log)

Thanks for cheak up.
Cheers,

**Comment by xforce** [comment]
Fixed in 0.8.2 https://github.com/xforce/anno1800-mod-loader/releases/tag/v0.8.2
